src/test/resources/ovs/v2/EventSchema.json

```diff
@@ -327,31 +327,7 @@
         },
         "eventLocation": {
           "description": "The location where the event takes place.",
-          "type": ["object","null"],
-          "default": "",
-          "properties": {
-            "locationName": {
-              "default": "",
-              "type": "string",
-              "maxLength": 100
-            },
-            "latitude": {
-              "default": "",
-              "type": "string",
-              "maxLength": 10
-            },
-            "longitude": {
-              "type": "string",
-              "maxLength": 11
-            },
-            "UNLocationCode": {
-              "type": "string",
-              "maxLength": 5
-            },
-            "address": {
-              "$ref": "#/definitions/address"
-            }
-          }
+          "$ref": "#/definitions/location"
         },
         "portCallServiceTypeCode": {
             "enum": ["PILO", "MOOR", "CRGO", "TOWG", "BUNK", "WSDP", null]
```

Comment:
```suggestion
            "enum": ["PILO", "MOOR", "CRGO", "TOWG", "BUNK", "WSDP", "LASH", "SAFE", "FAST", "GWAY", null]
```

Reply:
Don't know if you want these values in now or reject this if you want to add them later...